TransformationSystem/Client/TaskManager.py

```diff
@@ -420,15 +420,17 @@ def _handleDestination( self, paramsDict ):
     except KeyError:
       pass
 
-    if not self.destinationPlugin_o:
+    if self.destinationPlugin_o:
+      destinationPlugin_o = self.destinationPlugin_o
+    else:
       res = self.__generatePluginObject( self.destinationPlugin )
       if not res['OK']:
         self._logFatal( "Could not generate a destination plugin object" )
         return res
-      self.destinationPlugin_o = res['Value']
-      self.destinationPlugin_o.setParameters( paramsDict )
+      destinationPlugin_o = res['Value']
+      destinationPlugin_o.setParameters( paramsDict )
```

Comment:
Mmm you right I think. I will correct it but anyway right now this is not dangerous (self.destinationPlugin_o) is never set apart from the case of the unittest.